packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -131,24 +131,24 @@ export class OrgList implements vscode.Disposable {
     switch (selection) {
       case '$(plus) ' +
         nls.localize('force_auth_web_login_authorize_org_text'): {
-        vscode.commands.executeCommand('sfdx.force.auth.web.login');
-        return {
-          type: 'CONTINUE',
-          data: {}
-        };
-      }
+          vscode.commands.executeCommand('sfdx.force.auth.web.login');
+          return {
+            type: 'CONTINUE',
+            data: {}
+          };
+        }
       case '$(plus) ' +
         nls.localize('force_auth_web_login_authorize_dev_hub_text'): {
-        vscode.commands.executeCommand('sfdx.force.auth.dev.hub');
-        return { type: 'CONTINUE', data: {} };
-      }
+          vscode.commands.executeCommand('sfdx.force.auth.dev.hub');
+          return { type: 'CONTINUE', data: {} };
+        }
       case '$(plus) ' +
         nls.localize('force_org_create_default_scratch_org_text'): {
-        vscode.commands.executeCommand('sfdx.force.org.create');
-        return { type: 'CONTINUE', data: {} };
-      }
+          vscode.commands.executeCommand('sfdx.force.org.create');
+          return { type: 'CONTINUE', data: {} };
+        }
       default: {
-        const usernameOrAlias = selection.split(' ', 1);
+        const usernameOrAlias = selection.split(' - ', 1);
```

Comment:
This is the logic that was not accurately setting the alias name when it contained spaces.

Reply:
Does the CLI not require quotes when using spaces?

Reply:
Nope, they don't require quotes when using spaces.